druntime/src/core/stdc/stdatomic.d

```diff
@@ -169,9 +169,10 @@ void atomic_flag_clear_explicit_impl()(atomic_flag* obj, memory_order order)
             break;
 
         case memory_order.memory_order_acquire:
-            // What GCC does here is generate a relaxed order (no barriers)
-            // we'll copy it, although the D compiler should really be helping out at compile time.
-            atomicStore!(memory_order.memory_order_relaxed)(&obj.b, false);
+            // Iain requested this to default to seq_cst instead of relaxed,
```

Comment:
I don't think this line is necessary. Just start with "Ideally..." and be done with it.

At worst, use @ibuclaw handle so people from the future know who you are talking about.